examples/server/public/index.html

```diff
@@ -0,0 +1,459 @@
+                        </div>
+                        
+                        <div class="form-group">
+                            <label for="top_k">
+                                Top K: <span id="top_k-value" class="slider-value">40</span>
+                            </label>
+                            <div class="slider-container">
+                                <input type="range" id="top_k" min="1" max="100" step="1" value="40" class="slider">
+                            </div>
+                            <p class="hint">Number of most likely tokens to consider</p>
+                        </div>
+                        
+                        <div class="form-group">
+                            <label for="repetition_penalty">
+                                Repetition Penalty: <span id="repetition_penalty-value" class="slider-value">1.2</span>
+                            </label>
+                            <div class="slider-container">
+                                <input type="range" id="repetition_penalty" min="1" max="2" step="0.01" value="1.2" class="slider">
+                            </div>
+                            <p class="hint">Penalty for repeated tokens (higher = less repetition)</p>
+                        </div>
+                    </div>
+                </div>
+                
+                <button type="submit" id="submit-btn" class="submit-btn">
+                    <svg id="loading-spinner" class="spinner" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" style="display: none;">
+                        <circle cx="12" cy="12" r="10" stroke-opacity="0.25" stroke-width="4"></circle>
+                        <path d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z" fill="currentColor"></path>
+                    </svg>
+                    <span id="btn-text">Convert to Speech</span>
+                </button>
+                
+                <div id="error-msg" class="error-message" role="alert" aria-live="assertive"></div>
+            </form>
+            
+            <div id="audio-container" class="audio-container">
+                <audio controls id="audio-player" autoplay></audio>
+                <div class="audio-footer">
+                    <span>Generated audio</span>
+                    <a id="download-btn" class="download-btn">
+                        <svg width="14" height="14" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" style="margin-right: 4px;">
+                            <path d="M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4"></path>
+                            <polyline points="7 10 12 15 17 10"></polyline>
+                            <line x1="12" y1="15" x2="12" y2="3"></line>
+                        </svg>
+                        Download
```

Comment:
Do we need our own download button? On Chrome there's already a download menu item in the `<audio>`'s kebab menu. For Firefox, it can be right-clicked on desktop and long-pressed on mobile. I don't think it's necessary for us to have our own code for the download feature.